### ISSUE TITLE
Deprecating `area-web-frameworks` label

### ISSUE THREAD

**Issue by captainsafia** [dotnet/aspnetcore#57452]
Filing this issue to formally track deprecating the area-web-frameworks label. I attempted to do this last year but failing to retrain the labeling model so the label has mentioned.

In lieu of the `area-web-frameworks` model, we'd like to use the `area-minimal` and `area-mvc` labels to provide more granular tracking for issues that are impacting controller-based (`area-mvc`) and minimal (`area-minimal`) APIs. The intent is that we can more easily identify what issues are specific to controller-based APIs and minimal APIs for triage and prioritization.

There are some cases were an issue impacts both implementations (for example, an issue in our problem details writer). In that case, we can mark an issue with both labels.

Guide for labels is as follows:

- `area-minimal`: Issues related to minimal parameter binding, `TypedResults` implementation, RDG, native AoT for minimal
- `area-mvc`: Issues related to input/output formatters, model validation in MVC, model binding behavior in MVC, MVC conventions, MVC filters, IActionResults
- `area-mvc`, `area-minimal`: Issues related to problem details writer, OpenAPI implementation that impact both frameworks

I'm working through moving the ~20 issues we have in `area-web-frameworks` label to the more granular identifiers.

cc: @dotnet/aspnet-community-triagers for visibility and feedback

**Comment by gfoidl** [comment]
> * `area-mvc`, `area-minimal`: Issues related to problem details writer, OpenAPI implementation that impact both frameworks

For these both labels should be applied?
I'm asking because of the (unwritten) "one area label per issue" policy and to clarify the intention.

**Comment by captainsafia** [comment]
> For these both labels should be applied?

Yes, correct.

> I'm asking because of the (unwritten) "one area label per issue" policy and to clarify the intention.

I think historically the "one area label per issue" policy was in place to make sure that there was a single detected "owner" for a particular issue. In this case, the owners for the "area-minimal" and "area-mvc" are the same.

**Comment by captainsafia** [comment]
I tried re-training the labeler bot (see https://github.com/dotnet/issue-labeler) but the new model still seems to prefer area-web-frameworks for everything. It looks like we might have to relabel the closed issues as well to give the bot more accurate info...

**Comment by mkArtakMSFT** [comment]
Working with DDFun folks to grant us access to the storage so that we can upload the updated models.

**Comment by mkArtakMSFT** [comment]
@captainsafia isn't this fixed already?

**Comment by captainsafia** [comment]
@mkArtakMSFT No, we had to roll it back because updating the models seems to have broken the bot.

**Comment by mkArtakMSFT** [comment]
Ok, I thought there was more work done after that. No worries, I'm going to simply move this issue to .NET 10 planning for now
